meta-mel-support/recipes-bsp/96boards-tools/96boards-tools_0.7.bb

```diff
@@ -6,11 +6,12 @@ LICENSE = "GPLv2+"
 LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/GPL-2.0;md5=801f80980d171dd6425610833a22dbe6"
 
 SRCREV = "193f355823d9dc38f370759153ac950a2833b0e2"
-SRC_URI = "git://github.com/96boards/96boards-tools;branch=master;protocol=https"
+SRC_URI = "git://github.com/96boards/96boards-tools;branch=master;protocol=https \
+		file://resize-helper.sh"
```

Comment:
For the future, when using a template with placeholders, we should rename the file to .in, to clearly indicate that it's not a functioning script as is. But that's minor.